app/controllers/bathrooms_controller.rb

```diff
@@ -3,7 +3,11 @@ class BathroomsController < ApplicationController
 
   # GET /bathrooms or /bathrooms.json
   def index
-    @bathrooms=Bathroom.all
+    if params[:location].present?
+      @bathrooms = Bathroom.near(params[:location], params[:distance] || 15, order: :distance)
```

Comment:
Great setup for accessibility to the user! Good job thinking about the User's perspective - very intuitive.